### PR TITLE
organize scripts folder

### PR DESCRIPTION
**Pull request by justinmoon**
Closes https://github.com/fedimint/fedimint/issues/2134

### CODE REVIEW

File: scripts/final-checks.sh
Comment (justinmoon) [inline scripts/final-checks.sh:1]: I removed this file, doesn't seem like anyone uses it

File: scripts/gw-reload.sh
Comment (justinmoon) [inline scripts/gw-reload.sh:1]: I removed this file, don't think anyone uses it now that the cln extension is much simpler #fightingentropy

**Review by dpc — DISMISSED** [review]
I'm sure it will break something, but there's no other way really. :D

**Comment by dpc** [comment]
Probably missed adjusting flake source filters @justinmoon

File: scripts/gw-reload.sh
Comment (m1sterc001guy) [reply scripts/gw-reload.sh:1]: Now that the extension reliably dies with CLN, its easier to just kill/restart CLN itself.

**Comment by justinmoon** [comment]
@dpc passes ci now

**Review by elsirion — APPROVED** [review]

**Pull request merged**